docs/sourcemap-api.asciidoc

```diff
@@ -13,7 +13,7 @@ Send a `HTTP POST` request with the `Content-Type` header set to `multipart/form
 http(s)://{hostname}:{port}/assets/v1/sourcemaps
 ------------------------------------------------------------
 
-NOTE: The URL for uploading sourcemaps was previously `/v1/sourcemaps`. It still works, but is being deprecated in favor of `/assets/v1/sourcemaps`.
+NOTE: The URL for uploading sourcemaps was previously `/v1/rum/sourcemaps`. It was deprecated in v6.5 in favor of `/assets/v1/sourcemaps`.
```

Comment:
What do you think about mentioning that "the endpoint will be removed in version 7.0"?